adoc/advanced_topics_proxy_quickstart.adoc

```diff
@@ -101,58 +99,61 @@ in virtual environments, use the following settings for the virtual machine (VM)
 
 The following section will guide you through the installation and setup procedure.
 
-[IMPORTANT]
-.Registering Proxies
-====
 {productname} Proxy systems are registered as traditional clients or as Salt clients using a bootstrap script.
 A {susemgrproxy} can serve both Traditional and Salt clients.
-====
 
+[IMPORTANT]
+====
+First completely download the channels ({sle} 12 SP4) and then create the activation key.
+Only then you can select the correct child channels.
```

Comment:
Change the wording here so the most important thing comes first:
"Before you can select the correct child channels, ensure you have completely downloaded the channels for {sle} 12 SP4, and created the activation key."

Reply:
ok, but selecting/assigning channels take place while creating the activation key, I think I'd better say:
> Before you can select the correct child channels while creating the activation key, ensure you have completely downloaded the channels for {sle} 12 SP4.